src/Tooltip/js/tippy.js

```diff
@@ -323,7 +323,7 @@ class Tippy {
       if (
         isVisible(popper) ||
         !appendTo.contains(popper) ||
-        getComputedStyle(tooltip).opacity === '1'
+        getComputedStyle(tooltip).opacity !== '1'
```

Comment:
I have had a quick test, your fix seems to make the issue more than fix it, the tooltip in my usual use-case got stuck and never unmount out of DOM.

The original logic seems correct so, again, please help giving more clue / code snippet. Would very appreciate it.